.gitignore

```
@@ -48,3 +48,6 @@ install_manifest.txt
 compile_commands.json
 CTestTestfile.cmake
 _deps
+
+### Bazel build artifacts ###
+bazel-*
```

[Nit] This will ignore `bazel-*` in all directories, but you actually only care about the symlinks created at the root of the repo.

```suggestion
/bazel-*
```

---

Good point, thanks.